docs/guides/apache-sites-enabled.md

```diff
@@ -8,14 +8,14 @@
 
 # Apache Web Server Multi-Site Setup
 
-Rocky Linux has many ways for you to set up a web site. This is just one method, using Apache, and is designed for use as a multi-site setup on a single server. While this method is designed for multi-site servers, it can also act as a base configuration for a single site server as well. 
+Rocky Linux has many ways for you to set up a website. One way is using Apache, which is for a multi-site setup on a single server. While this method is for multi-site servers, it can also act as a base configuration for a single-site webserver. 
 
```

Comment:
This change is not correct: Apache is a way of creating web sites, the multi-site approach is one method within Apache to setup a set of web sites (which, by the way "web site" and "website" are both correct. Whichever is chosen should be consistent throughout the document), or to set up a single web site. By changing this sentence, you've removed the intended meaning.

Reply:
As concerns information, both of these statements are identical; show me where they are not. The problem is your second sentence is using a passive sense and is unnecessarily convoluted. I have not changed the intended meaning, only corrected the Grammar. I guess Chicago Manual of Style and all other standard formatting sources are incorrect when they say most usage now is the term website or Website. For reference see: https://writingexplained.org/website-or-web-site. The usage has changed over the years to now almost exclusively being used as one word. The reason is obvious. Website is a noun and normally two words are not one word until they are connected. Logically it should be web-site not web site and ultimately that is not important. I was just going for the better usage and that would be website or web-site, with or without upper case. Read the article I cite and you will see why this is the case.
Technically Apache is Web-server software; it holds the website but does not create the website. Humans create the website, Apache just serves or makes available the pages.

Reply:
“This is just one method” is grammatically incorrect because it is not clear what ‘This’ is referring ‘to’. That is why I changed it to ‘One way is using apache…’.

Now please explain how the meaning of the correction changes what the meaning of the paragraph was. And, specifically, please explain what the intended meaning was and how it has been changed.

Reply:
"One way is using Apache, which is for a multi-site setup on a single server."  That.. right there. This is not correct. Apache is not, by itself, used for multi-site setup... this particular configuration option, the one that is documented within, is. I'm fine with grammatical changes. I don't pretend to be an English teacher-not even close-but I have spent 21 years using Linux as a Systems Administrator and Network Administrattor AND writing documentation for the staff to follow for future uses. I'm retired now. 

My point is fix the grammar, but don't change the meaning of the document. If there really is a technical error (all of these procedures written by me were tested in the lab as they were written) then by all means fix those too.

Reply:
I also do not have  a problem with the "website" or "web site" argument. It just seemed a trivial change when both (at least at one time) were considered accurate.